server/controllers/spaceController.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response, NextFunction } from "express";
+import { Space } from "../services";
+
+export const getIssPosition = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<void> => {
+  try {
+    let serviceResponse = await Space.getIssPosition();
+    if (!serviceResponse) {
+      throw new Error("502 bad request.");
```

Comment:
This error is transforming any response from the ISS api that isn't a 200 into a 502, I don't think that's what we want to be doing here. What happens if you get a 404 or 400?

Reply:
The thing is, like in the documentation, we didn't consider a 404 or 400, just the 502 as the only possible case...?